Makefile

```
@@ -21,11 +21,13 @@ image: image-magneticod image-magneticow
 # Download dependencies
 ensure:
 	dep ensure -v
+	go get -u github.com/jteeuwen/go-bindata/...
```

Nice! But jteeuwen/go-bindata is unmaintained, so I use [github.com/kevinburke/go-bindata/](https://github.com/kevinburke/go-bindata/) instead.

```suggestion
	go get -u github.com/kevinburke/go-bindata/...
```